Neos.Flow/Classes/Configuration/RouteConfigurationProcessor.php

```diff
@@ -197,6 +197,12 @@ protected function buildSubRouteConfigurations(array $routesConfiguration, array
                         $mergedSubRouteConfiguration['defaults'][$key] = $this->replacePlaceholders($defaultValue, $variables);
                     }
                 }
+                foreach ($mergedSubRouteConfiguration['routeParts'] ?? [] as $routePartKey => $routePartValue) {
```

Comment:
Not sure I prefer this over a separate `isset()` condition, but it surely is concise. If we opt for this, I'd like to change the other occurences of `isset+foreach` accordingly (not necessarily in this PR)

Reply:
Yeah right, I'll stick to the `isset+foreach` mode for now and push a follow-up

Reply:
Done